sci-mathematics/mathics/mathics-1.0.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
```

Comment:
Please drop this line. It was recently decided to remove it from all ebuils, see https://bugs.gentoo.org/show_bug.cgi?id=611234. This will further retrigger repoman to check your commit. The build error you have commented on is actually not related to your ebuild.